Cargo.toml

```diff
@@ -8,13 +8,48 @@ authors = ["Chleba <chlebik@gmail.com>"]
 repository = "https://github.com/Chleba/netscanner"
 homepage = "https://github.com/Chleba/netscanner"
 
+[package.metadata.deb]
+maintainer = "Dominic Orsi <dominic.orsi@gmail.com>"
+depends = "$auto"
```

Comment:
Here from what I see in build logs we don't have specified auto dependencies, but we should have. Because netscanner depend on `iw` package (https://packages.debian.org/sid/iw) for Wi-Fi data. Without it, netscanner will crash.

Reply:
Updated, see commit [5a3019](https://github.com/Chleba/netscanner/pull/33/commits/5a301395f3004340d61affffcb9fb3c3f8d3cb66)